react-app/pomodoro-timer-ui/src/Components/TodoTab/Todo.jsx

```diff
@@ -29,28 +32,33 @@ export default function ({ todo, toggleComplete, removeTodo }) {
   function handleRemoveButton() {
     removeTodo(todo.id);
   }
+  
+  console.log("isActive",isActivePin)
```

Comment:
console log?

Reply:
can I do that within github, or do I go to the file in VS Code and recommit the changes?

Reply:
anyway you like, you just need to pull changes once merge into main branch

Reply:
yup, I got it